### PR TITLE
Changes to integrate with sensu-plugins-go-library

### PR DESCRIPTION
**Pull request by sayakju**
Update `sensu-influxdb-handler` to use new `sensu-plugins-go-library`

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,143 +1,129 @@
...
+			Argument:  dbName,
+			Shorthand: "d",
+			Default:   "",
+			Usage:     "the influxdb to send metrics to",
+			Value:     &config.DbName,
+		},
+		{
+			Path:      precision,
+			Argument:  precision,
+			Shorthand: "s",
```
Comment (nikkictl) [inline main.go]: There is no shorthand for the precision field.
```suggestion
			Shorthand: "",
```

File: main.go [hunk 2]
```diff
@@ -1,143 +1,129 @@
...
+			Default:   "",
```
Comment (nikkictl) [inline main.go]: `s` is the default value for the precision field.
```suggestion
			Default:   "s",
```

File: Gopkg.toml [hunk 3]
```diff
@@ -26,12 +26,8 @@
 
 
 [[constraint]]
-  name = "github.com/sensu/sensu-go"
```
Comment (nikkictl) [inline Gopkg.toml]: https://github.com/sensu/sensu-slack-handler/pull/19#discussion_r289935497

**Review by nikkictl — CHANGES_REQUESTED** [review]
Loving the cleanup and refactors.. a few comments.

File: Gopkg.toml (hunk 3 above)
Comment (sayakju) [reply Gopkg.toml]: sensu/sensu-pagerduty-handler#14 (comment)

**Comment by fguimond** [comment]
@nikkiki - The requested changes were implemented by @sayakju. Could you please review them when you have a chance? Thanks!

**Review by nikkictl — APPROVED** [review]

**Pull request merged**